test/js/source/tile_pyramid.test.js

```diff
@@ -313,8 +313,8 @@ test('TilePyramid#update', function(t) {
 
     t.test('retains parent tiles for pending children (wrapped)', function(t) {
         var transform = new Transform();
-        transform.width = 512;
-        transform.height = 512;
+        transform.width = 511;
+        transform.height = 511;
```

Comment:
@mourner:
> Hmm, I'd rather fix the tests to check row equality within a margin when perspective is set.

The tests are affected by which tiles are in the view's covering tiles. I didn't see an easy way to allow a certain margin of error in the output, so I just changed the size of the test view so that it wouldn't test a case where multiple possible results would be ok.